content/en/platform/corda/5.0-beta/operating/operating-tutorials/onboarding/mgm-onboarding.md

```diff
@@ -298,7 +298,7 @@ To set up the TLS key pair and certificate for the cluster:
    Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)} -Method Post -Uri "$API_URL/certificates/p2p/$TLS_KEY_ID" -Body (ConvertTo-Json @{
        x500Name = "CN=CordaOperator, C=GB, L=London"
        subjectAlternativeNames = @($P2P_GATEWAY_HOST)
-   }) > $WORK_DIR/request1.csr
+   }) > "$WORK_DIR"/request1.csr
```

Comment:
This seems to have been incorrectly added to the `PowerShell` tab instead of the `Bash` tab.